lightning/src/util/mod.rs

```diff
@@ -15,9 +15,11 @@ pub(crate) mod fuzz_wrappers;
 pub mod events;
 pub mod errors;
 pub mod ser;
+pub mod message_signing;
 
 pub(crate) mod byte_utils;
 pub(crate) mod chacha20;
+pub(crate) mod zbase32;
```

Comment:
Oops, the fuzztarget mode needs it, see CI.
```suggestion
#[cfg(feature = "fuzztarget")]
pub mod zbase32;
#[cfg(not(feature = "fuzztarget"))]
pub(crate) mod zbase32;
```

Reply:
Added it in cf384a3 and changed ``zbase32::{encode, decode}`` to pub so they can be used in fuzzing.